eva/udfs/udf_bootstrap_queries.py

```diff
@@ -127,13 +127,14 @@ def init_builtin_udfs(mode="debug"):
     """
     queries = [
         Fastrcnn_udf_query,
-        Mvit_udf_query,
```

Comment:
I think few test cases are failing due to removing this from the boostrap. Otherwise LGTM.